django_project/ford3/views/provider.py

```diff
@@ -72,17 +83,20 @@ def show_provider(request, provider_id):
     campus_query = Campus.objects.filter(provider__id=provider_id).annotate(
         campus_name=F('name'),
         campus_id=F('id'),
-        provider_name=F('provider__name')
+        provider_name=F('provider__name'),
+        provider_logo=F('provider__provider_logo')
     )
     campus_data = campus_query.values('name', 'id')
     provider_name = campus_query.values('provider_name')[0]['provider_name']
+    provider_logo = campus_query.values('provider_logo')[0]['provider_logo']
 
     form_data['campus_list'] = list(campus_data)
     form_data['provider_name'] = str(provider_name)
 
     context['form_data'] = form_data
     context['provider'] = {
         'campus': campus_data,
-        'id': provider_id
+        'id': provider_id,
     }
+    context['provider_logo'] = '/ford3' + settings.MEDIA_URL + provider_logo
```

Comment:
use url revere, or url join

Reply:
i added a variable called `path` in https://github.com/kartoza/ford3/blob/43e8a47df43ea24a915ce7eb8b245e637919076a/django_project/ford3/apps.py#L6